src/stubs/Test.stub

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace {{namespace}};
+
+use App\Http\Controllers\{{ucPluralModelName}}\{{modelName}}Controller;
+use App\Models\{{ucPluralModelName}}\{{modelName}};
+use Illuminate\Support\Facades\Route;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Shamaseen\Repository\Tests\TestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+class {{modelName}}Test extends TestCase
+{
+
+    use RefreshDatabase;
+
+    protected string $modelName = '{{modelName}}';
+    protected string  $routePath = "{{lcPluralModelName}}";
+
+       public function testIndex()
+       {
+           Route::get($this->routePath, [{{modelName}}Controller::class, 'index']);
```

Comment:
I think it is better to just keep the method body empty, or add a comment inside it, because:
1. this will warn the developer that there is an empty test with no assertion when he run the tests.
2. the developer will ALWAYS remove these lines, no body will keep these $this->assertTrue(true); null the Route inside test, so they are really unnecessary.
What do you think?